docs/PasswordManagement.md

```diff
@@ -0,0 +1,69 @@
+# Password Management
+
+- [Global kubernetes secret object](#global-kubernetes-secret-object)
+- [Default behavior of global kubernetes secret object](#default-behavior-of-global-kubernetes-secret-object)
+  - [Splunk Secret Tokens in the global secret object](#splunk-secret-tokens-in-the-global-secret-object)
+    - [HEC Token](#hec-token)
+    - [Default administrator password](#default-administrator-password)
+    - [Pass4symmkey](#pass4symmkey)
+    - [IDXC Pass4symmkey](#idxc-pass4symmkey)
+    - [SHC Pass4symmkey](#shc-pass4symmkey)
+- [Information for Splunk Enterprise administrator](#information-for-splunk-enterprise-administrator)
+- [Secrets on Docker Splunk](#secrets-on-docker-splunk)
+
+## Global kubernetes secret object
+A global kubernetes secret object acts as a common source of splunk secret tokens for all Splunk Enterprise CRs in a kubernetes namespace. It's name follows the format "splunk-<namespace>-secret" where <namespace> represents the namespace we are operating in. The contents of this object are volume mounted on all the pods within a kubernetes namespace. 
+
+This approach:
+  - Eliminates any mismatch between operator-generated secrets and admin provided secrets as all secrets are synced into a common object.
+  - Allows for dynamic adoption and modification of secrets.
+
+## Default behavior of global kubernetes secret object
+
+For a given namespace upon the creation of the first Splunk Enterprise CR, the operator checks for the existence of a global kubernetes secret object:
+
+- If the object does not exist,
+    - Creates the object
```

Comment:
maybe mention the name of secret too, like "Created the object with name `splunk-<namespace>-secret` " .

Reply:
Done.